### PR TITLE
Completed object-manipulation-2

### PR DESCRIPTION
**Pull request by joshangeles**
image

### CODE REVIEW

File: object-manipulation-2/src/defaults.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* exported defaults */
+/*
+   Go through each key found in source and:
+   Check if that key value matches any key value found in target
+   If it does NOT match any key then...
+   Add that key to target object and assign its value as the value associated with that key in source object BUT
+   If the values found are different but the keys exist in both object do not assign the source value to the target object
+*/
+function defaults(target, source) {
+  var targetKeys = Object.keys(target);
```
Comment (codingcodymiller) [inline object-manipulation-2/src/defaults.js:10]: Try to do this without turning the object into an array 🤓

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: object-manipulation-2/src/defaults.js (hunk 1 above)
Comment (joshangeles) [reply object-manipulation-2/src/defaults.js]: image
image
Ended up shortening it by quite a bit!

File: object-manipulation-2/src/omit.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/* exported omit */
+/*  Go through each property found in input source object
+    Check if each found property includes any of the input keys -use includes() method to simply matching input keys to found property
+    If it does not match any of the input keys, push it into an empty storage object
+    Return the filled storage object
+   */
+function omit(source, keys) {
+  var omitted = {};
+  for (var property in source) {
+    if (keys.includes(property) === false) {
```
Comment (RobertGardner) [inline object-manipulation-2/src/omit.js:10]: Since `includes` returns a boolean, you could just write `if (!keys.includes(property))`

**Review by RobertGardner — APPROVED** [review]
Well done.

**Pull request merged**